tests/basic/pthread_detach/README.md

```diff
@@ -0,0 +1,181 @@
+# Preliminaries
+## Thread identities
+A thread has 3 identities:
+- Linux process context - `task_struct`
+- LKL arch specific process context - `thread_info`
+- LKL host thread context - `lthread`
```

Comment:
lthread is SGX-LKL-specific terminology, it might be worth explicitly stating that this is called the host thread in LKL terminology.
```suggestion
- LKL host thread context, implemented in SGX-LKL by `lthread`s.
```